mock/py/mockbuild/config.py

```diff
@@ -294,9 +294,18 @@ def setup_default_config_opts():
     config_opts['system_yum_command'] = '/usr/bin/yum'
     config_opts['yum_install_command'] = 'install yum yum-utils'
     config_opts['yum_builddep_command'] = '/usr/bin/yum-builddep'
-    config_opts['dnf_command'] = '/usr/bin/dnf'
     config_opts['system_dnf_command'] = '/usr/bin/dnf'
+
+    config_opts['dnf_command'] = '/usr/bin/dnf'
+    config_opts['dnf_common_opts'] = ['--setopt=deltarpm=False', '--allowerasing']
     config_opts['dnf_install_command'] = 'install dnf dnf-plugins-core'
+    config_opts['dnf_disable_plugins'] = ['local', 'spacewalk', 'versionlock']
+
+    config_opts['dnf5_command'] = '/usr/bin/dnf5'
+    config_opts['dnf5_common_opts'] = ['--setopt=deltarpm=False', '--allowerasing']
+    config_opts['dnf5_install_command'] = 'install dnf5 dnf5-plugins'
+    config_opts['dnf5_disable_plugins'] = []
```

Comment:
Can you add this to site-defaults.cfg?

Reply:
I added (also others) and I bit reorganized the package-manager-related documentation.  Hope that it is helpful.